contracts/BaseStrategy.sol

```diff
@@ -1,10 +1,9 @@
 // SPDX-License-Identifier: GPL-3.0
-pragma solidity >=0.6.0 <0.7.0;
-pragma experimental ABIEncoderV2;
+pragma solidity 0.8.2;
```

Comment:
Use the entire `0.8.x` range here, these contracts are meant to be pinned downstream